packages/client/src/api/session.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+export const session = axios.create({
+	baseURL: 'https://ya-praktikum.tech/api/v2'
+});
+
+session.interceptors.response.use(
+	response => response,
```

Comment:
Вкусовщина, но я бы всегда оборачивал аргументы стрелочных функций в скобки. Ментально так проще воспринимать что происходит в строке.
```suggestion
	(response) => response,
```